wdl-lint/src/v1/import_placement.rs

```diff
@@ -0,0 +1,118 @@
+//! A lint rule for import placements.
+
+use wdl_ast::v1::ImportStatement;
+use wdl_ast::v1::StructDefinition;
+use wdl_ast::v1::TaskDefinition;
+use wdl_ast::v1::Visitor;
+use wdl_ast::v1::WorkflowDefinition;
+use wdl_ast::AstNode;
+use wdl_ast::Diagnostic;
+use wdl_ast::Diagnostics;
+use wdl_ast::Span;
+use wdl_ast::ToSpan;
+use wdl_ast::VisitReason;
+
+use super::Rule;
+use crate::Tag;
+use crate::TagSet;
+
+/// The identifier for the import placement rule.
+const ID: &str = "ImportPlacement";
+
+/// Creates a "misplaced import" diagnostic.
+fn misplaced_import(span: Span) -> Diagnostic {
+    Diagnostic::warning("misplaced import")
+        .with_rule(ID)
+        .with_highlight(span)
+        .with_fix("move this import so that it comes directly after the version statement")
+}
+
+/// Detects incorrect import placements.
+#[derive(Debug, Clone, Copy)]
+pub struct ImportPlacementRule;
+
+impl Rule for ImportPlacementRule {
+    fn id(&self) -> &'static str {
+        ID
+    }
+
+    fn description(&self) -> &'static str {
+        "Ensures that imports are placed immediately following the version statement."
```

Comment:
Same question for `immediately` as I had for `directly`.

Reply:
How about `Ensures that imports are placed between the version statement and any document items`?

Reply:
That sounds good 👍